Connecting to server on port 3002...
Connected to server on port 3002
Something went wrong running your bot: 'str' object has no attribute 'position'
Traceback (most recent call last):
File "/Users/sanjilee/development/lebrawnjame_mm30/main.py", line 245, in serve
response = strategy.steer_input(planes)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/Users/sanjilee/development/lebrawnjame_mm30/strategy/sanji_strategy.py", line 82, in steer_input
if abs(enemy.position.y - friendly.position.y) < 5:
^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'position'. Did you mean: 'partition'?
Something went wrong running your bot: 'str' object has no attribute 'position'
Traceback (most recent call last):
File "/Users/sanjilee/development/lebrawnjame_mm30/main.py", line 245, in serve
response = strategy.steer_input(planes)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/Users/sanjilee/development/lebrawnjame_mm30/strategy/sanji_strategy.py", line 82, in steer_input
if abs(enemy.position.y - friendly.position.y) < 5:
^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'position'. Did you mean: 'partition'?
Something went wrong running your bot: 'str' object has no attribute 'position'
Traceback (most recent call last):
File "/Users/sanjilee/development/lebrawnjame_mm30/main.py", line 245, in serve
response = strategy.steer_input(planes)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/Users/sanjilee/development/lebrawnjame_mm30/strategy/sanji_strategy.py", line 82, in steer_input
if abs(enemy.position.y - friendly.position.y) < 5:
^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'position'. Did you mean: 'partition'?
Your bot failed to respond 3 times in a row (is your bot broken?) and was disconnected